client/post-editor/editor-ground-control/test/index.jsx

```diff
@@ -181,7 +181,7 @@ describe( 'EditorGroundControl', function() {
 				/>
 			).instance();
 
-			expect( tree.getPrimaryButtonLabel() ).to.equal( 'Publish' );
+			expect( tree.getPrimaryButtonLabel() ).to.equal( 'Update' );
```

Comment:
So it may be that we want to detect whether the `savedPost` is published already, showing "Update" if it is, and "Publish" otherwise?